js/src/forum/components/PostsUserPage.js

```diff
@@ -59,6 +60,15 @@ export default class PostsUserPage extends UserPage {
       );
     }
 
+    if (this.posts.length === 0 && !this.loading) {
+      const text = app.translator.trans('core.forum.user.posts_empty_text');
+      return (
+        <div className="PostsUserPage">
+          {Placeholder.component({text})}
+        </div>
+      );
+    }
```

Comment:
This entire if statement could probably go to the very top of the method - we don't need to set the `footer` variable if we're not using it anyway...

Reply:
@franzliedke Hm, true. It would only shave off a millisecond or two in performance, though, if at all. The function is not running all the time... unless a madman with a loop & `m.redraw` comes along 😂

Reply:
1736fe3f585b551c7b858b1c0f49a2b5b83664a8

Reply:
It's not so much about performance as about clarity. Thanks, Toby!